src/components/filter/style.css

```diff
@@ -1,3 +1,89 @@
+* {
+  box-sizing: border-box;
+}
+
 :host {
-  display: block;
+  display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	position: relative;
+	width: 100%;
+}
+
+:host > smoothly-input {
+	margin-bottom: 0.5em;
+}
+
+.hidden {
+	display: flex;
+	width: 0px;
+	opacity: 0;
+}
+
+.container {
+	width: inherit;
+	position: relative;
+	z-index: 3;
+	margin-bottom: 1em;
+	opacity: 1;
+	transition: opacity 100ms ease-in-out;
+}
+
+.container:after {
+	content:"";
+	height: 100%;
+	width: 100%;
+	border: 1px solid rgb(var(--text-color, var(--smoothly-default-contrast)));
+	border-radius: 0.25em;
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+
+/* arrow */
+.container:before {
+	content:"";
+	width: 0.6em;
+	height: 0.6em;
+	transform: rotate(45deg);
+	border-top: 1px solid rgb(var(--text-color, var(--smoothly-default-contrast)));
+	border-left: 1px solid rgb(var(--text-color, var(--smoothly-default-contrast)));
+	background-color: white;
```

Comment:
I think Simon wants us to only use smoothly colors, even if it's black or white.